clients/gtest/blas1_gtest.cpp

```diff
@@ -211,18 +223,7 @@ TEST_P(blas1_gtest, axpy_float)
     // while the tuple is non-intuitive.
     Arguments arg = setup_blas1_arguments( GetParam() );
     rocblas_status status = testing_axpy<float>( arg );
-    // if not success, then the input argument is problematic, so detect the error message
-    if(status != rocblas_status_success){
-        if( arg.N < 0 ){
-            EXPECT_EQ(rocblas_status_invalid_size, status);
-        }
-        else if( arg.incx < 0){
-            EXPECT_EQ(rocblas_status_invalid_size, status);
-        }
-        else if( arg.incy < 0){
-            EXPECT_EQ(rocblas_status_invalid_size, status);
-        }
-    }
+    EXPECT_EQ(rocblas_status_success, status);
```

Comment:
Does this reduce test precision?  Looks like the old code tried to decipher that the returned error code was a particular value.

Reply:
Maybe you meant to use your new macro CHECK_ROCBLAS_ERROR()?